opflexagent/utils/bridge_managers/ovs_manager.py

```diff
@@ -31,12 +32,28 @@ class OvsManager(bridge_manager_base.BridgeManagerBase,
     def __init__(self):
         super(OvsManager, self).__init__()
 
-    def initialize(self, host, ovs_config, opflex_conf):
+    def initialize(self, host, opflex_conf, agent_state):
+        try:
+            ovs_config = cfg.CONF.OVS
+        except cfg.NoSuchOptError:
+            raise
+        try:
+            bridge_mappings = helpers.parse_mappings(
+                ovs_config.bridge_mappings)
+        except ValueError as e:
+            raise ValueError(_("Parsing bridge_mappings failed: %s.") % e)
         self.int_br_device_count = 0
         self.int_br = ovs_lib.OVSBridge(ovs_config.integration_bridge)
         self.fabric_br = ovs_lib.OVSBridge(opflex_conf.fabric_bridge)
         self.setup_integration_bridge()
-        return self
+        agent_state['bridge_mappings'] = bridge_mappings
+        agent_state['datapath_type'] = ovs_config.datapath_type
+        agent_state['vhostuser_socket_dir'] = ovs_config.vhostuser_socket_dir
```

Comment:
Does the OVS bridge manager need vhostuser_socket_dir?

Reply:
Yes to support ovs-dpdk.